src/taxman.py

```diff
@@ -475,6 +477,54 @@ def _evaluate_taxation_GERMANY(self, op: tr.Operation) -> None:
                 )
                 self.tax_report_entries.append(report_entry)
 
+        elif isinstance(op, tr.MarginFee):
+            # Fees for margin trading
+            self.remove_from_balance(op)
+            if in_tax_year(op):
+                taxation_type = "Kapitaleinkünfte aus Margin Trading"
+                report_entry = tr.MarginReportEntry(
+                    platform=op.platform,
+                    amount=op.change,
+                    coin=op.coin,
+                    utc_time=op.utc_time,
+                    interest_in_fiat=-self.price_data.get_cost(op),
+                    taxation_type=taxation_type,
+                    remark=op.remark,
+                )
+                self.tax_report_entries.append(report_entry)
+
+        elif isinstance(op, tr.MarginGain):
+            # Gains from margin trading
+            self.add_to_balance(op)
+            if in_tax_year(op):
+                taxation_type = "Kapitaleinkünfte aus Margin Trading"
+                report_entry = tr.MarginReportEntry(
+                    platform=op.platform,
+                    amount=op.change,
+                    coin=op.coin,
+                    utc_time=op.utc_time,
+                    interest_in_fiat=self.price_data.get_cost(op),
+                    taxation_type=taxation_type,
+                    remark=op.remark,
+                )
+                self.tax_report_entries.append(report_entry)
+
+        elif isinstance(op, tr.MarginLoss):
+            # Losses from margin trading
+            self.remove_from_balance(op)
+            if in_tax_year(op):
+                taxation_type = "Kapitaleinkünfte aus Margin Trading"
+                report_entry = tr.MarginReportEntry(
+                    platform=op.platform,
+                    amount=op.change,
```

Comment:
Shouldn't this be negative as we are giving it aways?

Reply:
Yes, is fixed now.